app/test/testHealthCheck.js

```diff
@@ -0,0 +1,39 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const {app} = require('../service');
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Heartbeat Tests', function() {
+  it('should return 200', (done) => {
+    chai.request(app)
+      .get('/__heartbeat__')
+      .end((err, res) => {
+        res.should.have.status(200);
+        done();
+      });
+  });
+});
+
+describe('LBHeartbeat Tests', function() {
+  it('should return 200', (done) => {
+    chai.request(app)
+      .get('/__lbheartbeat__')
+      .end((err, res) => {
+        res.should.have.status(200);
+        done();
+      });
+  });
+});
+
+describe('Version Tests', function() {
+  it('should return 200 and version data', (done) => {
+    chai.request(app)
+      .get('/__version__')
+      .end((err, res) => {
+        res.should.have.status(200);
```

Comment:
Do you want to assert that the `/__version__` endpoint returns an object with the following keys:
- commit
- version
- source

Otherwise the `it()` statement of _"... and version data"_ seems a bit misleading.